pandas/io/formats/format.py

```diff
@@ -1802,6 +1802,10 @@ def _save_chunk(self, start_i, end_i):
                                         date_format=self.date_format,
                                         quoting=self.quoting)
 
+        # Flatten ix when nlevels = 1 and ix is array of tuples.
```

Comment:
this is not the place to fix it. it should be handled in the MI formatter itself.

Reply:
I agree.  168833489d22ac79e1e214a70f93096c388c9852